fetch.py

```diff
@@ -0,0 +1,93 @@
+import typer
+from rdflib.namespace import RDF, RDFS, VOID, XSD, SOSA, DCTERMS, PROV, TIME
+from rdflib import Graph, Namespace, URIRef, Literal, BNode
+
+app = typer.Typer()
+
+
+@app.command()
+def fetch(filename: str):
+
+    g = Graph()
+
+    SHACL = Namespace("http://www.w3.org/ns/shacl#")
+    g.bind("sh", SHACL)
+
+    g.parse(filename)
+
+    q_delete_bnodes = """
+        PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+        PREFIX sh: <http://www.w3.org/ns/shacl#>
+        delete {
+            ?s sh:xone ?node .
+            ?node rdf:rest ?item .
+            ?item rdf:first ?head ;
+                rdf:rest ?tail .
+        }
+        where {
+            ?s sh:xone ?node .
+            ?node rdf:rest* ?item .
+            ?item rdf:first ?head ;
+                rdf:rest ?tail .
+        }
+    """
+
+    q_delete_values = """
+    PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+    PREFIX sh: <http://www.w3.org/ns/shacl#>
+    delete {
+        ?s sh:property ?property .
+        ?property sh:path rdf:value ;
+        sh:hasValue ?value .
+    }
+    where {
+        ?s sh:property ?property .
+        ?property sh:path rdf:value ;
+        sh:hasValue ?value .
+    }
+    """
```

Comment:
Is there a reason this is a separate query? We should only be deleting where `?s` is part of the `sh:xone`. This query deletes all `?s` where `?s` has an `sh:property`.